packages/common/stateInfo.ts

```diff
@@ -92,11 +92,9 @@ interface SignatureBaseInfo extends BaseInfo {
 
 export const arizonaParty = ['Democratic Party', 'Republican Party', 'Non-Partisan', 'Green Party (Pima County only)'] as const
 export type ArizonaParty = (typeof arizonaParty)[number]
-export const isArizonaParty = (x: string | null): x is ArizonaParty => arizonaParty.includes(x as ArizonaParty)
```

Comment:
We don't need these functions anymore since we got type safety from controlled `<Select/>`

Reply:
Thanks for clearing this!